.cirrus.yml

```diff
@@ -894,14 +870,55 @@ freebsd12_task:
     - git submodule update --recursive --init
 
   configure_script:
-    - ./configure --generator=Ninja --enable-ccache --prefix=/opt/spicy --with-flex=/usr/local --with-bison=/usr/local --with-zeek=/usr/local --enable-werror --build-zeek-plugin=yes
+    - ./configure --generator=Ninja --enable-ccache --prefix=/opt/spicy --with-flex=/usr/local --with-bison=/usr/local --enable-werror
   build_script:
-    - ninja -j4 -C build install install-spicy-analyzers
+    - ninja -j4 -C build install
   test_install_script:
-    - SPICY_INSTALLATION_DIRECTORY=/opt/spicy make -C tests test-all-install
+    - SPICY_INSTALLATION_DIRECTORY=/opt/spicy make -C tests test
   packaging_script:
     - ninja -C build package
 
   packages_artifacts:
     path: build/spicy*.tar.gz
     type: application/gzip
+
+# zeek_plugin_task:
```

Comment:
Let's remove this.

Reply:
It was just a draft, that's why it was commented out. Revised and activated now.